### ISSUE TITLE
Cannot assign craft\elements\Address to property webdna\commerce\bulkpricing\adjusters\Tax

### ISSUE THREAD

**Issue by john-henry** [webdna/commerce-bulk-pricing#16]
Just installed latest on Craft Craft CMS 4.1.4.1 and Craft Commerce 4.1.x-dev

I have not added the field yet to any product and get this when viewing frontend

`Cannot assign craft\elements\Address to property webdna\commerce\bulkpricing\adjusters\Tax::$_address of type ?craft\commerce\models\Address`

Created field and added to Product layout and same error

Happens when I'm logged in

I am admin and have 3 addresses in my account 

https://share.johnhenry.ie/mXuxGPZN

**Comment by robzor** [comment]
Ah I guess this is because address is now a native craft field/property as well as the user no longer being separate?

**Comment by john-henry** [comment]
Yeah likely. I'm available for testing anything you want as we are on same time zone

**Comment by robzor** [comment]
I'm just an interested third party but the guys at webdna are pretty responsive. I actually know you from the EE world, long time no see :)

**Comment by john-henry** [comment]
Ah yes of course! Electric Putty days. Feel so old :)

**Comment by robzor** [comment]
I wasn't experiencing this error but I realised I didn't have any addresses associated with users (still very early in development) and once I added some and tried to manipulate them (for instance, copy billing address to shipping address in an order) the error appears.

**Comment by samuelbirch** [comment]
Thanks @john-henry and @robzor , we'll get looking into this.

**Comment by samuelbirch** [comment]
v2.0.0-beta.3 should fix this.

**Comment by robzor** [comment]
Gotta love a one line fix! Thanks @samuelbirch for smashing these fixes out :)
